sweep/walletsweep.go

```diff
@@ -232,7 +232,10 @@ func CraftSweepAllTx(feeRate lnwallet.SatPerKWeight, blockHeight uint32,
 		// The rest of the items will be populated internally within
 		// the sweeper via the witness generation function.
 		signDesc := &input.SignDescriptor{
-			Output:   outputInfo,
+			Output: &wire.TxOut{
+				PkScript: outputInfo.PkScript,
+				Value:    int64(output.Value),
```

Comment:
Is there a reason not to use `Value` from `outputInfo`?

Reply:
We actually don't even need to make the `FetchInputInfo` call here because we already have all the information required from `ListUnspentWitness`.

Reply:
Ok, even better